src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.no_cov_crate.txt

```diff
@@ -6,8 +6,8 @@
     6|       |    println!("called but not covered");
     7|       |}
     8|       |
-    9|       |#[no_coverage]
-   10|       |fn do_not_add_coverage_2() {
+    9|       |fn do_not_add_coverage_2() {
+   10|       |    #![no_coverage]
```

Comment:
I think maybe you didn't mean to include this change in your PR? The placement and syntax are incorrect (if I'm not mistaken).

Reply:
This was intentional, just wanted to confirm that `#[no_coverage]` works as an inner attribute as well, just like all other attrs should in theory.